co-log-benchmark/Main.hs

```diff
@@ -106,3 +109,13 @@ timeProcess n = do
       runProcess_ cfg
   t' <- getCurrentTime
   pure $ t' `diffUTCTime` t
+
+-- |  Function that takes list of pairs - benchmark name
+-- and result and generates markdown table.
+genTable pairs = intercalate "\n" rows
+  where
+    pairs' = ("Benchmark", "Result") : ("---","---:") : fmap (second show) pairs
+    f = (Max).length
+    (fMax,sMax) = coerce $ foldMap (bimap f f)  pairs'
+    rows = fmap (\(b,r) -> b <> replicate (fMax - (length b)) ' ' <> " | "
```

Comment:
I'd probably pull out another subfunction for the `replicate` part, in order to make rows a bit more clear
type signatures for these subfunctions wouldn't hurt